app/routes/_layout+/kits.$kitId.manage-assets.tsx

```diff
@@ -488,8 +542,18 @@ export default function ManageAssetsInKit() {
 
       {/* Footer of the modal */}
       <footer className="item-center -mx-6 flex justify-between border-t px-6 pt-3">
-        <div className="flex items-center font-medium">
-          {selectedAssets.length} assets selected
+        <div className="flex items-center gap-2 font-medium">
+          <p>
+            {hasSelectedAll ? selectedAssets.length - 1 : selectedAssets.length}{" "}
+            assets selected
+          </p>
+          <Button
+            variant="secondary"
+            className="px-2 py-1 text-sm font-normal"
+            onClick={handleSelectAll}
+          >
+            {hasSelectedAll ? "Clear all" : "Select all"}
```

Comment:
Clicking "Select all" seems to only select items in current page. Should work like on the other places where it can actually select all assets.